tests/test_es_doc_creator.py

```diff
@@ -366,6 +367,7 @@ def __init__(self) -> None:
         "int": 123,
         "list": [1, 2, 3, 4],
         "none": None,
+        "non_ecs_compliant_attribute_name_": True,
```

Comment:
we should consider making sure that the attribute doesnt start or end with an underscore